### PR TITLE
FIWARELabNodesSLA script added

### PR DESCRIPTION
**Pull request by K4s14**

### CODE REVIEW

File: scripts/FIWARELabNodesSLA/README.org [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#+TITLE:     FIWARE Lab Nodes SLA
+#+AUTHOR:    Katarzyna Di Meo
+#+DESCRIPTION: Python app for nodes' SLA computing
+#+KEYWORDS:  python, sla, nodes, fiware
+#+LANGUAGE:  en
+
+#+html: <p align="left"><a href="http://www.apache.org/licenses/LICENSE-2.0"><img src="https://camo.githubusercontent.com/651d047a62d409c28b23517069c652af8b8c429c/68747470733a2f2f636f636f61706f642d6261646765732e6865726f6b756170702e636f6d2f6c2f526573744b69742f62616467652e706e67" /></a></p>
+
+* General description
+  The nodes, that are funded by the FI-NEXT project, have to assure a certain level of quality defined through a Service Level Agreement (SLA). One of the conditions that must be satisfied is the services availability on the node that must be above the 95% threshold.\\
```
Comment (chicco785) [inline scripts/FIWARELabNodesSLA/README.org]: The nodes taking part to FIWARE Lab have to assure a certain level of quality defined through a Service Level Agreement (SLA). One of the conditions that must be satisfied is the services availability on the node that must be above the 95% threshold.\\

File: scripts/FIWARELabNodesSLA/README.org [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  A set of OpenStack services, called FIWARE Lab node services, runs on each of the FIWARE Lab nodes. The measurement of the availability of those services is managed through the use of the FIWARE Lab monitoring system tool. This FIWARE Lab monitoring system collects all the monitoring measurements from each FIWARE Lab node. These informations are centrally stored, aggregated and available through the FIWARE monitoring API for both real-time raw data and aggregated historical data.
+  
+  FIWARE relies on the above pipeline to calculate the FIWARE Lab node overall service availability analysing the uptime status of:
```
Comment (chicco785) [inline scripts/FIWARELabNodesSLA/README.org]: FIWARE Lab Admins rely on the above pipeline to calculate the FIWARE Lab node overall service availability analysing the uptime status of:

**Review by chicco785 — CHANGES_REQUESTED** [review]
Hi @K4s14 i made few specific comments, additionally:
* use MD syntax for README.org -> README.md (to be uniform)
* apply an MD linter to check formatting and limit lines to 80 chars.

**Review by K4s14 — COMMENTED** [review]
I think the README file should be in the correct format now. I changed also the text you suggested.

File: scripts/FIWARELabNodesSLA/README.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+This Python application is used to compute the averaged uptime of the services 
+of each region by calling the API 
+[Services4Region](https://federationmonitoring.docs.apiary.io/#reference/service/services4region)
+of the FIWARE monitoring APIs.
+
+The application can be run by each node by setting some optional arguments and 
+a configuration file:
+
+1) optional arguments:
```
Comment (chicco785) [inline scripts/FIWARELabNodesSLA/README.md]: 1.

File: scripts/FIWARELabNodesSLA/README.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+    - -h, --help: shows the help message and exit
```
Comment (chicco785) [inline scripts/FIWARELabNodesSLA/README.md]: use code syntax for the parameters

File: scripts/FIWARELabNodesSLA/README.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+      previous month is considered.
+    - -r REGION_ID, --region-id REGION_ID: sets the region to analyse. Setting 
+      "ALL", all regions will be analysed
+    - -w, --weekend: if specified, the week ends will be considered
+    - -l, --log: if specified, prints the averaged uptime of the services for 
+      each day
+    - -v, --version: if specified, prints the version of this SLA script
+    
+2) configuration file:
+
```
Comment (chicco785) [inline scripts/FIWARELabNodesSLA/README.md]: 1.

File: scripts/FIWARELabNodesSLA/README.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  ```bash
```
Comment (chicco785) [inline scripts/FIWARELabNodesSLA/README.md]: increase indent of the code so that is aligned with the list

**Review by chicco785 — APPROVED** [review]
minor fixes, i believe that after that we can merge

**Pull request merged**